### DISCUSSION TITLE
exception handling

### DISCUSSION THREAD

**Discussion by anshuld1995** [community/community#20675] [Other Feature Feedback, Questions, & Ideas] [answered]
"//table[@class='wikitable sortable jquery-tablesorter']/tbody/tr/td[1]" 
so in this last tbody/tr/td present and i want to what is this in exception handling plz somebody clear me this

**Comment by mohsin-riad** [comment, marked as answer]
hello @anshuld1995 ✋🏻
An exception is an error that happens at the time of execution of a program. However, while running a program, any programming language generates an exception that should be handled to avoid your program from crashing. we can solve any exception by Handling it via ```try``` & ```except```.

for example: ```Python```
```
str = "//table[@Class='wikitable sortable jquery-tablesorter']/tbody/tr/td[1]"

try:  #if '/tbody/tr/td[1]' doesn't exist this book will work as you already defined
    #try to perform your work here

except:  #if '/tbody/tr/td[1]' exist, it will process it via below logic
    str = str.split('/')[:3]
    temp = "".join(s for s in str)
    print(temp)  #you can just check your desired one here.  
```

if you find my answer helpful, please feel free to `mark as answered`
